### PR TITLE
Table labels

### PR DESCRIPTION
**Pull request by codymccoy**

### CODE REVIEW

File: ViewModel/Watershed/TerrainBrowserVM.cs [hunk 1]
```diff
@@ -18,6 +19,7 @@ public class TerrainBrowserVM:Editors.BaseEditorVM
         private const string FLT = ".flt";
         private const string TIF = ".tif";
 
+        private string _PreviousName;
```
Comment (RyanM-RMA) [inline ViewModel/Watershed/TerrainBrowserVM.cs:22]: This variable doesn't seem to be used beyond setting it.

**Review by RyanM-RMA — CHANGES_REQUESTED** [review]
Got some changes for you

File: ViewModel/FrequencyRelationships/AnalyticalFrequencyEditorVM.cs [hunk 2]
```diff
@@ -126,6 +126,11 @@ public ObservableCollection<FlowDoubleWrapper> GraphicalFlows
         #region Constructors
         public AnalyticalFrequencyEditorVM(ComputeComponentVM defaultCurve,  EditorActionManager actionManager) : base(defaultCurve, actionManager)
         {
+            _Mean = DefaultCurveData.LP3Mean;
+            _StDev = DefaultCurveData.LP3StDev;
+            _Skew = DefaultCurveData.LP3Skew;
+            _POR = DefaultCurveData.LP3POR;
```
[the remark this answers is not in the document]
Comment (codymccoy) [reply ViewModel/FrequencyRelationships/AnalyticalFrequencyEditorVM.cs:132]: why?

File: ViewModel/Watershed/TerrainBrowserVM.cs [hunk 3]
```diff
@@ -30,7 +32,7 @@ public string TerrainPath
             get { return _TerrainPath; }
             set
             {
-                _TerrainPath = value;NotifyPropertyChanged();
+                _TerrainPath = value; NotifyPropertyChanged();
```
[the remark this answers is not in the document]
Comment (codymccoy) [reply ViewModel/Watershed/TerrainBrowserVM.cs:35]: it should be on one line with the "set"

File: ViewModel/FrequencyRelationships/AnalyticalFrequencyEditorVM.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (RyanM-RMA) [reply ViewModel/FrequencyRelationships/AnalyticalFrequencyEditorVM.cs:132]: Sorry I put this on the wrong line.  _POR is also initialized when it's declared above.

[the remark this answers is not in the document]
Comment (codymccoy) [reply ViewModel/FrequencyRelationships/AnalyticalFrequencyEditorVM.cs:132]: i got rid of the assignment at the top of the class and kept the assignment in the ctor.

**Review by RyanM-RMA — APPROVED** [review]

**Pull request merged**